tests/chainer_tests/functions_tests/connection_tests/test_convolution_3d.py

```diff
@@ -0,0 +1,130 @@
+import unittest
+
+import mock
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer.functions.connection import convolution_3d
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+from chainer.utils import conv
+
+def _triplet(x):
+    if hasattr(x, '__getitem__'):
+        return x
+    return x, x, x
+
+
+def _asfortranarray(x):
```

Comment:
asfortanarray is already implemented in both `numpy` and `cupy`. So we do not have to implement `asfortranarray` by yourself.